api/nodes/permissions.py

```diff
@@ -128,9 +128,9 @@ def has_object_permission(self, request, view, obj):
                 return False
             pointer_nodes = []
             for pointer in request.data.get('data', []):
-                node = Node.load(pointer['id'])
+                node = AbstractNode.load(pointer['id'])
                 if not node or node.is_collection:
-                    raise exceptions.NotFound(detail='Node with id "{}" was not found'.format(pointer['id']))
+                    raise exceptions.NotFound(detail='AbstractNode with id "{}" was not found'.format(pointer['id']))
```

Comment:
I don't think we want to use the term "AbstractNode" in user-facing error messages; probably better to keep this as is.